fluent-syntax/Cargo.toml

```diff
@@ -37,7 +37,8 @@ serde_json.workspace = true
 glob = "0.3"
 
 [features]
-default = []
+default = ["spans"]
```

Comment:
I would not enable this by default. I think it's is an optional feature needed for editors/LSPs.